packages/charts/src/chart_types/xy_chart/state/chart_state.tsx

```diff
@@ -104,7 +104,7 @@ export class XYAxisChartState implements InternalChartState {
   }
 
   getLegendItemsLabels(globalState: GlobalChartState) {
-    return getLegendItemsLabelsSelector(globalState);
+    return getLegendItemsLabelsSelector(globalState as unknown as never);
```

Comment:
It looks like this selector was made useless in #2355 but never removed.

We are really averse to using `as` in charts. In this case, I think we can just remove this selector as it's doing nothing. The only reason the selector types are complaining is because the selector input is empty.

We should also delete this selector here...

https://github.com/elastic/elastic-charts/blob/bfb0d28f77794f7e35fa54381c05e21ef1ccaf58/packages/charts/src/chart_types/xy_chart/state/selectors/get_legend_items_labels.ts#L13

```suggestion
    return [];
```

Reply:
Thanks for the context, that's helpful! I took the opportunity and consolidated the available constants a bit (`EMPTY_...`) in [2b75cf9](https://github.com/elastic/elastic-charts/pull/2607/commits/2b75cf95270bdeafb941db6b2fbe41da5c5a4dd8).

Reply:
Deleted the selector in 6aff1ed.